src/Symfony/Bundle/FrameworkBundle/Resources/config/routing.xml

```diff
@@ -66,6 +67,7 @@
                 <argument key="matcher_base_class">%router.options.matcher_base_class%</argument>
                 <argument key="matcher_dumper_class">%router.options.matcher_dumper_class%</argument>
                 <argument key="matcher_cache_class">%router.options.matcher.cache_class%</argument>
+                <argument key="route_collection_cache_file">%router.options.route_collection.cache_file%</argument>
```

Comment:
@nicolas-grekas, I think I got it! I've found the docs about caching based on resources.